config/locales/devise.et.yml

```diff
@@ -59,6 +59,13 @@ et:
       send_paranoid_instructions: "Kui sinu e-posti aadress on meie andmebaasis, saadetakse  mõne minuti jooksul sinu e-posti aadressile link salasõna taastamiseks."
       updated: "Salasõna on muudetud. Oled sisse logitud."
       updated_not_active: "Salasõna on muudetud."
+      new:
+        reset_button: "Saada parooli muutmise juhised"
+      edit:
+        new_password: "Uus parool"
+        min_characters: "Vähemalt %{min_chars} märki"
+        confirm_new_password: "Sisesta uus parool uuesti"
+        change_password: "Muuda mu parool"
```

Comment:
Quotes are needed here?

Reply:
Not at all, just used them solely to support the overall translations style.